gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file |
-|  saturday | Enum | **Required** | The saturday field contains a binary value that indicates whether the service is valid for all Saturdays. |
-|   |  | | A value of **1** indicates that service is available for all Saturdays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Saturdays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file. |
-|  sunday | Enum | **Required** | The sunday field contains a binary value that indicates whether the service is valid for all Sundays. |
-|   |  | | A value of **1** indicates that service is available for all Sundays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Sundays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file. |
-|  start_date | Date | **Required** | The **start_date** field contains the start date for the service.  The start_date field's value should be in YYYYMMDD format. |
-|  end_date | Date | **Required** | The **end_date** field contains the end date for the service. This date is included in the service interval.  The **end_date** field's value should be in YYYYMMDD format. |
+|  service_id | ID | **Required** | Identifies a set of dates when service is available for one or more routes. |
+|  monday | Enum | **Required** | Indicates whether the service has been added or removed for all Mondays in the date range specified by the `start_date` and `end_date` fields. Note that exceptions for particular may be listed in [calendar_dates.txt](#calendar_datestxt). Valid values are: |
+|   |  | | `1` - Service is available for all Mondays in the date range. |
+|   |  | | `0` - Service is not available for Mondays in the date range. |
+|  tuesday | Enum | **Required** | Functions in the same way as `monday` except applies to Tuesdays |
+|  wednesday | Enum | **Required** | Functions in the same way as `monday` except applies to Wednesdays  |
+|  thursday | Enum | **Required** | Functions in the same way as `monday` except applies to Thursdays  |
+|  friday | Enum | **Required** | Functions in the same way as `monday` except applies to Fridays  |
+|  saturday | Enum | **Required** | Functions in the same way as `monday` except applies to Saturdays. |
+|  sunday | Enum | **Required** | Functions in the same way as `monday` except applies to Sundays. |
+|  start_date | Date | **Required** | Start date for the service interval. |
```

Comment:
If you add the definition of service day somewhere, you could improve this by saying: "Start *service day* for the service interval".